### ISSUE TITLE
Add full working example or link to tutorial with working example

### ISSUE THREAD

**Issue by clecap** [nathancahill/split#220]
Thanx for the great library, but I was unable to get it working from the "examples" in the readme. 

The readme does not provide a full working example but just code snippets. It still is not clear which parameter I missed, but I did not get a working sample through hours. :-(

For me https://redstapler.co/split-js-tutorial-website-split-screen-effect/ was very helpful since they provided a simple, minimal working example which was running out of the box. 

Suggest you either provide your own working examples (as in fully complete, minimal stand alone example) or add a link to the above site, since they are having one.

**Comment by abyabrkal** [comment]
> Thanx for the great library, but I was unable to get it working from the "examples" in the readme.
> 
> The readme does not provide a full working example but just code snippets. It still is not clear which parameter I missed, but I did not get a working sample through hours. :-(
> 
> For me https://redstapler.co/split-js-tutorial-website-split-screen-effect/ was very helpful since they provided a simple, minimal working example which was running out of the box.
> 
> Suggest you either provide your own working examples (as in fully complete, minimal stand alone example) or add a link to the above site, since they are having one.

Yes, absolutely right. I was struggling with same and above link was really helpful.

**Comment by abyabrkal** [comment]
Yes, absolutely right. I was struggling with the same and above link was really helpful.

**Comment by vasyl-shumskyi** [comment]
same here. simple jsfiddle would be awesome!

**Comment by mohanen** [comment]
yea, like wtf. I know its foss but still i spent more time hitting my head in getting it done

**Comment by mohanen** [comment]
> Thanx for the great library, but I was unable to get it working from the "examples" in the readme.
> 
> The readme does not provide a full working example but just code snippets. It still is not clear which parameter I missed, but I did not get a working sample through hours. :-(
> 
> For me https://redstapler.co/split-js-tutorial-website-split-screen-effect/ was very helpful since they provided a simple, minimal working example which was running out of the box.
> 
> Suggest you either provide your own working examples (as in fully complete, minimal stand alone example) or add a link to the above site, since they are having one.

@clecap That link helped me, i got it working in a minute.

**Comment by bryik** [comment]
Here is [a small CodePen demo](https://codepen.io/bryik/pen/wvaeorY).

The first example in [the readme](https://split.js.org/) appears broken because the elements being split are empty divs and the gutter has no height. The example works once you [add some dimensions and color](https://codepen.io/bryik/pen/bGdRBKz). It would be better if the example was "copy and paste"-ready (also flex is better than float IMO).
